earth_enterprise/src/fusion/gepublish/gepublishmanagerhelper.h

```diff
@@ -20,12 +20,11 @@
 #ifndef GEO_EARTH_ENTERPRISE_SRC_FUSION_GEPUBLISH_GEPUBLISHMANAGERHELPER_H_
 #define GEO_EARTH_ENTERPRISE_SRC_FUSION_GEPUBLISH_GEPUBLISHMANAGERHELPER_H_
 
-#include <stddef.h>
+#include <Python.h>
+#include <cstddef>
 
-#include <python2.7/Python.h>  // Note: should be first in include list.
-
-#include <string>
-#include <vector>
+#include "string"
```

Comment:
Q: Why is this changed to `"string"` and `"vector"`? Aren't we including the standard library headers here?